src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -188,407 +140,17 @@ impl BDA {
-        f.debug_struct("StaticHeader")
-            .field("blkdev_size", &self.blkdev_size)
-            .field("pool_uuid", &self.pool_uuid.to_simple_ref())
-            .field("dev_uuid", &self.dev_uuid.to_simple_ref())
-            .field("mda_size", &self.mda_size)
-            .field("reserved_size", &self.reserved_size)
-            .field("flags", &self.flags)
-            .field("initialization_time", &self.initialization_time)
-            .finish()
-    }
-}
-
 #[cfg(test)]
 mod tests {
     use std::io::Cursor;
 
-    use proptest::{collection::vec, num, option, prelude::BoxedStrategy, strategy::Strategy};
-    use uuid::Uuid;
-
-    use devicemapper::{Bytes, Sectors, IEC};
+    use proptest::{collection::vec, num};
 
     use super::*;
-
-    /// Return a static header with random block device and MDA size.
-    /// The block device is less than the minimum, for efficiency in testing.
-    fn random_static_header(blkdev_size: u64, mda_size_factor: u32) -> StaticHeader {
-        let pool_uuid = Uuid::new_v4();
-        let dev_uuid = Uuid::new_v4();
-        let mda_size = MDADataSize::new(
-            MDADataSize::default().bytes() + Bytes(u64::from(mda_size_factor * 4)),
-        )
-        .region_size()
-        .mda_size();
-        let blkdev_size = (Bytes(IEC::Mi) + Sectors(blkdev_size).bytes()).sectors();
-        StaticHeader::new(
-            pool_uuid,
-            dev_uuid,
-            mda_size,
-            BlockdevSize::new(blkdev_size),
-            Utc::now().timestamp() as u64,
-        )
-    }
-
-    /// Make a static header strategy
-    fn static_header_strategy() -> BoxedStrategy<StaticHeader> {
-        (0..64u64, 0..64u32)
-            .prop_map(|(b, m)| random_static_header(b, m))
-            .boxed()
-    }
-
-    proptest! {
-        #[test]
-        /// Construct an arbitrary StaticHeader object.
-        /// Verify that the "memory buffer" is unowned.
-        /// Initialize a static header.
-        /// Verify that Stratis buffer validates.
-        /// Wipe the static header.
-        /// Verify that the buffer is again unowned.
-        fn test_ownership(ref sh in static_header_strategy()) {
-            let buf_size = *sh.mda_size.sectors().bytes() as usize + bytes!(static_header_size::STATIC_HEADER_SECTORS);
-            let mut buf = Cursor::new(vec![0; buf_size]);
-            prop_assert!(StaticHeader::setup(&mut buf).unwrap().is_none());
-
-            sh.write(&mut buf, MetadataLocation::Both).unwrap();
-
-            prop_assert!(StaticHeader::setup(&mut buf)
-                         .unwrap()
-                         .map(|new_sh| new_sh.pool_uuid == sh.pool_uuid && new_sh.dev_uuid == sh.dev_uuid)
-                         .unwrap_or(false));
-
-            StaticHeader::wipe(&mut buf).unwrap();
-            prop_assert!(StaticHeader::setup(&mut buf).unwrap().is_none());
-        }
-    }
+    use crate::engine::strat_engine::backstore::metadata::{
```

Comment:
Please reverse the order of the ```use super::*``` and the ```use crate...```. We want the most specific import, the one from this module, last.